scripts/calcRD.sh

```diff
@@ -12,11 +12,15 @@ count=$3
 
 numerator=$(echo $frd $count| awk ' { printf "%0.2f\n", ($1 * $2); } ')
 
+readsToSample=$(echo $numerator $ord| awk ' { printf "%0.2f\n", ($1 / $2); } ')
+
+echo $readsToSample
+
+
+#This calcuates how many reads to random sample
+#bc worked on erisone but not in the docker, so switched to awk above 
+#
 #$(bc <<< "$frd*$count")
```

Comment:
Would be good to clean up commented-out lines like this - since you're using github for versioning they'll still be available in your past versions if you need to reference them, so no need to keep them as clutter in the final version